argo/apps/templates/securebanking-rs.yaml

```diff
@@ -20,6 +20,17 @@ spec:
         value: "mongodb-{{ .Release.Name }}"
       - name: deployment.imageOverride.tag
         value: {{ .Values.rs.tag }}
+      values: |
+        springConfig:
+          testdata:
+            userAccountIds:
+              psu4test:
+                - sortCode: "012332"
+                  accountNumber: "43245676"
+                - sortCode: "012332"
+                  accountNumber: "54312390"
+                - sortCode: "334412"
+                  accountNumber: "30187862"
```

Comment:
There's a limitation in ArgoCD which means maps cannot be passed as parameter overrides, therefore values have to be embedded directly here.

In the Release Environments this will be much cleaner, as we aren't using Argo and can therefore just drop these into the umbrella chart's values file.